.circleci/config.yml

```diff
@@ -472,7 +472,7 @@ jobs:
             export JAVA_HOME=/usr/lib/jvm/jre-1.8.0-openjdk
             export HADOOP_ROOT_LOGGER="WARN,DRFA"
             export LIBHDFS3_CONF=$(pwd)/.circleci/hdfs-client.xml
-            export PATH=~/adapter-deps/install/bin:~/adapter-deps/install/hadoop-2.10.1/bin:${PATH}
+            export PATH=~/adapter-deps/install/bin:~/adapter-deps/install/bin/hadoop-2.10.1/:${PATH}
```

Comment:
@Mionsz should we just remove `~/adapter-deps/install/hadoop-2.10.1/` instead of fixing this?

Reply:
@majetideepak Yes I think you can even remove hadoop task as now I see that hadoop is being downloaded and built when setup-circleci.sh is run for Docker container creation: [./scripts/setup-circleci.sh](https://github.com/facebookincubator/velox/blob/main/scripts/setup-circleci.sh) lines 73-77
```bash
wget_and_untar https://github.com/fmtlib/fmt/archive/8.0.0.tar.gz fmt &
#  wget_and_untar https://github.com/ericniebler/range-v3/archive/0.11.0.tar.gz ranges-v3 &
wget_and_untar https://archive.apache.org/dist/hadoop/common/hadoop-2.10.1/hadoop-2.10.1.tar.gz hadoop

wait  # For cmake and source downloads to complete.

cp -a hadoop /usr/local/

# Build & install.
(
  cd lzo
  ./configure --prefix=/usr --enable-shared --disable-static --docdir=/usr/share/doc/lzo-2.10
  make "-j$(nproc)"
  make install
)
```

Reply:
Can you try removing the "Install Hadoop dependency" run and check?

Reply:
We probably still need `yum -y install java-1.8.0-openjdk`